samples/SocialSample/Startup.cs

```diff
@@ -136,7 +136,7 @@ k web
                         JObject user = JObject.Parse(text);
 
                         var identity = new ClaimsIdentity(
-                            context.Options.AuthenticationType,
+                            context.Options.AuthenticationScheme,
```

Comment:
We never smush principals though - just append to the existing one on context (and remove identity on context)